lib/generated/intl/messages_en.dart

```diff
@@ -24,6 +24,12 @@ class MessageLookup extends MessageLookupByLibrary {
   static Map<String, Function> _notInlinedMessages(_) => <String, Function>{
```

Comment:
Is it possible to exclude these generated files from the repo?

Reply:
Yep, I think so. I'll need to add generation to our Github action, I will add it as part of this the translations tickets. Already added it to the description